packages/core/src/animation/Animator.ts

```diff
@@ -135,7 +137,7 @@ export class Animator extends Component {
     manuallyTransition.duration = normalizedTransitionDuration;
     manuallyTransition.offset = normalizedTimeOffset;
     manuallyTransition.destinationState = state;
-    this._crossFadeByTransition(manuallyTransition, layerIndex);
+    this._crossFadeByTransition(manuallyTransition, layerIndex, true);
```

Comment:
Why not split the logic like play? There is no relationship between `crossFadeByTransition` and `update`